app/views/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= stylesheet_link_tag "fonts", "application", media: "all", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+  </head>
+
+  <body class="<%= params[:lookbook][:display][:body_classes] %>">
```

Comment:
Should we be using `dig` here in case one of these is empty?

Also, let's convert to `<%= tag.body class: "" do %>` so we don't have interpolated ERB inside of a String.